AbsoluteFill: add ref forwarding

satelllte opened the pull request:
<!---
  Please do:
  - read CONTRIBUTING.md before sending a pull request
  - link issues that the PR is affecting (e.g #123)
  - try to make the test suite pass
  - add documentation
  - document potential tradeoffs in this PR.
-->

This is rarely used but I think it's still worth to fill this gap since the component is highly reusable

JonnyBurger reviewed (dismissed):
Super useful! In fact I wanted to do this yesterday as well and had to work around it!
Thanks a lot for implementing this 🙌

JonnyBurger commented:
Also implemented it for `<Sequence>` and added docs to it btw!

satelllte commented:
> Super useful! In fact I wanted to do this yesterday as well and had to work around it!
Thanks a lot for implementing this 🙌

> Also implemented it for <Sequence> and added docs to it btw!

Nice, thanks @JonnyBurger. Waiting for them both to be live 🚀 🛰️
